src/ui/api/utils.go

```diff
@@ -377,7 +379,7 @@ func diffRepos(reposInRegistry []string, reposInDB []string) ([]string, []string
 	for i < len(reposInRegistry) {
 		repoInR = reposInRegistry[i]
 		i++
-		exist, err := projectExists(repoInR)
+		exist, err := projectExists(pm, repoInR)
```

Comment:
The function utils.ParseRepository(repo) has two return values, so we can not put it as the parameter of pm.Exist().